frontend/components/LastUpdated.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import * as timeago from "timeago.js";
+import { useState, useEffect } from "react";
+
+import styles from "../styles/LastUpdated.module.css";
+
+const LastUpdated = ({ update }) => {
+  const [time, setTime] = useState();
+
+  useEffect(() => {
+    setTime(timeago.format(new Date(update)));
+  }, []);
```

Comment:
Linter says this is missing. It probably doesn't make any practical difference, especially in such a small component, but might as well be correct.
```suggestion
  }, [update]);
```